src/components/Dashboard/TodoList.vue

```diff
@@ -83,14 +84,35 @@
                   </div>
                 </div>
 
-                <div v-if="isTypeCorrection(task) && isStatusDraft(task)" class="todo-subtitle">
+                <div v-else-if="isTypeCorrection(task) && isStatusDraft(task)" class="todo-subtitle">
                   <div>DRAFT</div>
                   <v-btn x-small icon class="expand-btn">
                     <v-icon>mdi-message-reply</v-icon>
                   </v-btn>
                   Detail{{task.comments.length > 1 ? "s" : ""}} ({{task.comments.length}})
                 </div>
 
+                 <div v-else-if="isBcolError(task)" class="todo-subtitle">
+                  <div>FILING PENDING</div>
+                  <div class="vert-pipe"></div>
+                  <div class="payment-status" v-if="inProcessFiling === task.id">
+                    PROCESSING...
```

Comment:
Although this is labelled "pending", I think it's not the same as a true pending, where the BE may still be doing work, so the FE retries 10X for an update (which is where the "PROCESSING..." comes in. In this case I think there is no processing state.

Reply:
Fixed.